src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,64 @@
+package com.aca.homework.week4.snake;
+
+public class Snake {
+    private static int snakeSize;
+    private int x;
+    private int y;
+    private char symbol;
+    private int size;
```

Comment:
the size variable name is not understandable. It might be for instance indexFromTheHead